submissions/NadiaVorontsova/hooli-style-popup/index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>HTML-CSS-Popup</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="header__menu">
+                <ul class="header__list">
+                    <li><a href="#" class="header__link link">Mail</a></li>
+                    <li><a href="#" class="header__link link">Images</a></li>
+                    <li>
+                        <input type="checkbox" id="popup_checkbox" class="header__popup__checkbox" checked>
```

Comment:
[Watch the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide checkboxes in the right way.

Reply:
I fixed it;) please review my changes